test/e2e/kamsuite/kamsuite.go

```diff
@@ -238,31 +235,32 @@ func createRepository(repo string) error {
 	return nil
 }
 
-func loginArgoAPIServerLogin() error {
+func loginArgoAPIServerLogin() bool {
```

Comment:
@varshab1210 Why do we return bool here? Without the errors, it's very difficult to debug what went wrong